test/services/ReviewServiceSpec.scala

```diff
@@ -542,6 +589,10 @@ class ReviewServiceSpec extends UnitSpec with MockFactory with ReviewData with A
             .getById("validId")
             .returns(Future.successful(Right(factCheckProcess)))
 
+          MockApprovalProcessReviewRepository
+            .updateReview("validId", approvalProcess.version, ReviewTypeFactCheck, statusChange2iReviewInfo.userId, statusChange2iReviewInfo.status)
+            .returns(Future.successful(Right(())))
+
           MockApprovalRepository
             .changeStatus("validId", StatusWithDesignerForUpdate, "user id")
             .returns(Future.successful(expected))
```

Comment:
Test at line 610. It would better to test update review is not called as this is invoked before changeStatus.

Reply:
ok, will update in next commit